.ci/azure/linux.yml

```diff
@@ -141,7 +139,7 @@ jobs:
     displayName: 'Java tests'
 
   - script: |
-      python3 -m pip install $(INSTALL_DIR)/tools/*.whl
+      python3 -m pip install $(INSTALL_DIR)/tools/*.whl cython
```

Comment:
Better to replace `requirenemts_test` to https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/src/compatibility/openvino/requirements-dev.txt